src/components/Loader/Loader.tsx

```diff
@@ -0,0 +1,15 @@
+import classNames from 'classnames';
+import React from 'react';
+
+type Props = {
+  isActive: boolean,
+};
+
+export const Loader: React.FC<Props> = ({ isActive }) => {
+  return (
+    <div className={classNames('modal overlay', { 'is-active': isActive })}>
+      <div className="modal-background has-background-white-ter" />
+      <div className="loader" />
```

Comment:
Consider adding an aria-label to the loader element for better accessibility. ♿

Reply:
thanks, done